dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_config_panel_fields.tsx

```diff
@@ -29,6 +29,12 @@ import {
   DataConfigPanelFieldProps,
 } from '../../../../../../../../common/types/explorer';
 
+const HIDE_ADD_BUTTON_VIZ_TYPES = [
```

Comment:
Would this bee too many lines to be an inclusion filter instead of an elimination filter?

There are only 20 types or so arent there?

Not sure which way carries better comprehension?

Reply:
Here, I think the elimination filter that we are using is a better comprehension because we are trying to find if the string is not an element in the array. Please let me know if you have any better thoughts on this.